services/datamanager/builtin/sync_test.go

```diff
@@ -105,44 +110,49 @@ func TestSyncEnabled(t *testing.T) {
 			updatedSvcConfig.CaptureDisabled = false
 			updatedSvcConfig.ScheduledSyncDisabled = tc.newServiceDisableStatus
 			updatedSvcConfig.CaptureDir = tmpDir
-			updatedSvcConfig.SyncIntervalMins = fiftyMillis
+			updatedSvcConfig.SyncIntervalMins = syncIntervalMins
 
 			err = dmsvc.Update(context.Background(), cfg)
 			test.That(t, err, test.ShouldBeNil)
 
-			// Let run for a second, then change status.
-			time.Sleep(syncTime)
+			// Drain any requests that were already sent before Update returned.
+			for len(mockClient.succesfulDCRequests) > 0 {
+				<-mockClient.succesfulDCRequests
+			}
+			var receivedReqTwo bool
```

Comment:
Should this also be `sentReqTwo`? Also very nit but maybe `sentSecondReq` would be clearer. Just because often variables are named like `variable1` and `variable2` so it might not be obvious that you mean we sent a second request.

Reply:
Yup 100% I just forgot to update this - changed this from `received` to `sent` and changed the name to `sentRequestAfterUpdate` to make it a bit more clear what this variable is meant to capture